modules/cli/test-scripts/streams-tests.sh

```diff
@@ -0,0 +1,250 @@
+    --name $maven_prop_name \
+    --value http://points-to-my-maven-repo.example.org \
+    --bootstrap $bootstrap \
+    --log -"
+
+    local cmd3="${BINARY_LOCATION} properties set --namespace framework \
+    --name $testCatalog_prop_name \
+    --value http://points-to-my-test-catalog.example.org \
+    --bootstrap $bootstrap \
+    --log -"
+
+    local cmd4="${BINARY_LOCATION} properties set --namespace framework \
+    --name $obr_prop_name \
+    --value mvn:myorg/myartifact/0.0.1/obr \
+    --bootstrap $bootstrap \
+    --log -"
+
+    info "Command is: $cmd1"
+    info "Command is: $cmd2"
+    info "Command is: $cmd3"
+    info "Command is: $cmd4"
+
+    $cmd1
+    $cmd2
+    $cmd3
+    $cmd4
+    local rc=$?
+
+    # We expect a return code of 0 because this is a properly formed properties set command.
+    if [[ "${rc}" != "0" ]]; then 
+        error "Failed to create property with name and value used."
+        return 1
+    fi
+    
+    return 0
+}
+
+#-----------------------------------------------------------------------------------------
+# Tests
+#----------------------------------------------------------------------------------------- 
+function streams_create {
+    h2 "Performing properties set with name and value parameter used: create..."
+
+    set -o pipefail # Fail everything if anything in the pipeline fails. Else we are just checking the 'tee' return code.
+    
+    stream_name="mystream"
+    configure_stream "$stream_name" "$bootstrap"
+    rc=$?
+    
+    if [[ "${rc}" != "0" ]]; then 
+        error "Failed to configure stream properties."
+        exit 1
+    fi
+
+    # check that stream has been created
+    cmd="${BINARY_LOCATION} streams get \
+    --name $stream_name \
+    --bootstrap $bootstrap \
+    --log -"
+
+    info "Command is: $cmd"
+
+    output_file="$ORIGINAL_DIR/temp/streams-get-output.txt"
+    $cmd | tee $output_file
+    if [[ "${rc}" != "0" ]]; then 
+        error "Failed to get stream with name used: command failed."
+        exit 1
+    fi
+
+    # Check that the previous properties set created a stream property
+    cat $output_file | grep "$stream_name" -q
+
+    rc=$?
+    # We expect a return code of 0 because this is a properly formed streams get command.
+    if [[ "${rc}" != "0" ]]; then 
+        error "Failed to create stream with name and value used."
+        exit 1
+    fi
+
+    # Check that the previous properties set created a stream, with the correct values
+    cat $output_file | grep "Galasa" -q -E
+    cat $output_file | grep "mystream" -q -E
+    cat $output_file | grep "enabled" -q -E
+
+    rc=$?
+    # We expect a return code of 0 because this is a properly formed properties get command.
+    if [[ "${rc}" != "0" ]]; then 
+        error "Stream successfully created, but value incorrect."
+        exit 1
+    fi
+
+    success "Stream set with name and value used seems to have been created correctly."
+}
```

Comment:
I think we need more tests to check that `streams get` works as expected.

We could add a test for a simple `streams get` without passing in a `--name`, a test for `streams get --name`, pass in `--format yaml`, a test for `streams get` with a `--name` for a stream that doesn't exist?